circle.yml

```diff
@@ -20,3 +20,4 @@ deployment:
     branch: master
     commands:
       - NODE_ENV=prod npm run build
+      - npm publish
```

Comment:
@donnapep Should this be `npm prepublish`? I noticed in package.json that's the name of the script.